host-configs/LLNL/lassen-cuda-11-base.cmake

```diff
@@ -0,0 +1,15 @@
+include(${CMAKE_CURRENT_LIST_DIR}/lassen-base.cmake)
+
+# Cuda options
+set(ENABLE_CUDA ON CACHE BOOL "")
+set(CUDA_TOOLKIT_ROOT_DIR /usr/tce/packages/cuda/cuda-11.2.0 CACHE STRING "")
```

Comment:
```suggestion
set(CUDA_TOOLKIT_ROOT_DIR /usr/tce/packages/cuda/cuda-11.6.1 CACHE STRING "")
```

Reply:
do we have 11.6 now? I ll update it on the docker image too then.

Reply:
Current lassen cuda module list:
![image](https://github.com/GEOS-DEV/LvArray/assets/4522899/a1eb4220-6fe6-47a1-9962-35b8bd7bb3ac)

Reply:
so should we use the latest cuda11 ?

Reply:
Looking at the release notes they did some work in 11.7 and 11.8 to reduce both host and device memory footprints, so unless we have a compelling reason not to, I would say go for 11.8 yeah.